examples/example_data/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.encrypted
```

Comment:
Do you know which examples are spitting out these files, and can we fix them to clean up after themselves or have them write to a temporary directory? I hate messy tests. :)

Reply:
Yes, the `file_streaming.cpp` example generates `*.encrypted` and `*.decrypted` files alongside the original target file which is passed as an command-line argument. I'll look into how best to clean them up after tests are run.